multichain-testing/tools/random.ts

```diff
@@ -0,0 +1,6 @@
+export function makeRandomDigits(digits = 2n) {
+  if (digits < 1n) throw new Error('digits must be positive');
+  const maxValue = Math.pow(10, Number(digits)) - 1;
+  const num = Math.floor(Math.random() * (maxValue + 1));
```

Comment:
`Math.random` is ambient authority. There's only 1 caller of this function. Please move it to the test script or inline it so that we're not exporting functions that close over ambient authority.

Reply:
This seemed like too much to inline - I made `randomNumber` a parameter and put the `Math.random()` call in the test file.